src/main/java/org/folio/inventory/dataimport/handlers/actions/InstanceUpdateDelegate.java

```diff
@@ -43,7 +44,7 @@ public Future<Instance> handle(Map<String, String> eventPayload, Record marcReco
     try {
       JsonObject mappingRules = new JsonObject(eventPayload.get(MAPPING_RULES_KEY));
       MappingParameters mappingParameters = new JsonObject(eventPayload.get(MAPPING_PARAMS_KEY)).mapTo(MappingParameters.class);
-
+      AdditionalFieldsUtil.normalize035(marcRecord);
```

Comment:
looks like, we need to add this method call to ReplaceInstanceEventHandler for being called when instance is updated during data import